src/Form/ApplicationRequest/ApplicationRequestListFilterType.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace AppBundle\Form\ApplicationRequest;
+
+use AppBundle\ApplicationRequest\Filter\ListFilter;
+use AppBundle\Entity\ApplicationRequest\Theme;
+use AppBundle\Form\GenderType;
+use AppBundle\Repository\ApplicationRequest\ThemeRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ApplicationRequestListFilterType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('firstName', TextType::class, [
+                'required' => false,
+            ])
+            ->add('lastName', TextType::class, [
+                'required' => false,
+            ])
+            ->add('gender', GenderType::class, [
+                'placeholder' => 'Tous',
+                'expanded' => true,
+                'required' => false,
+            ])
+            ->add('isAdherent', CheckboxType::class, [
+                'required' => false,
+            ])
+            ->add('theme', EntityType::class, [
+                'class' => Theme::class,
+                'required' => false,
+                'placeholder' => 'common.all',
+                'query_builder' => static function (ThemeRepository $themeRepository) {
+                    return $themeRepository->createDisplayabledQueryBuilder();
+                },
+            ])
+        ;
+
+        if ($options['extended']) {
+            $builder
+                ->add('isInMyTeam', ChoiceType::class, [
+                    'choices' => [
+                        'Oui' => 1,
+                        'Non' => 2,
+                        'Déjà pris dans une autre ville' => 3,
+                    ],
+                    'placeholder' => 'common.all',
+                    'required' => false,
+                ])
+                ->add('tag', TagType::class, [
+                    'multiple' => false,
+                    'placeholder' => 'common.all',
+                    'required' => false,
+                ])
+            ;
+        }
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setDefaults([
+                'data_class' => ListFilter::class,
+                'extended' => false,
+            ])
+            ->setAllowedTypes('extended', ['boolean'])
```

Comment:
`->setAllowedTypes('extended', 'bool')` ?

Reply:
even if it works well thanks to Type aliases I'll change it:
 
https://github.com/symfony/symfony/blob/bcc6d31ce439aeb625fba3756a79b7bfa7076c9c/src/Symfony/Component/OptionsResolver/OptionsResolver.php#L106-L110